locale/en/knowledge/intermediate/how-to-log.md

```diff
@@ -57,17 +57,19 @@ Here is an example of setting up a `winston` logger.  This example includes most
 
       require('winston-riak').Riak;
       require('winston-mongo').Mongo;
+      require('winston-couchdb').Couchdb;
+      
       var logger = new (winston.Logger)({
         transports: [
-          new winston.transports.Console()
-          new winston.transports.File({ filename: 'path/to/all-logs.log' })
-          new winston.transports.Couchdb({ 'host': 'localhost', 'db': 'logs' })
-          new winston.transports.Riak({ bucket: 'logs' })
-          new winston.transports.MongoDB({ db: 'db', level: 'info'})
-        ]
+          new winston.transports.Console(),
+          new winston.transports.File({ filename: 'path/to/all-logs.log' }),
+          new winston.transports.Couchdb({ 'host': 'localhost', 'db': 'logs' }),
+          new winston.transports.Riak({ bucket: 'logs' }),
+          new winston.transports.MongoDB({ db: 'db', level: 'info'}),
```

Comment:
I don't like arrays written like this `[ 1,2,3, ]` but if other reviewers are ok with the trailing commas I'll live with it,